h2o-bindings/bin/gen_java.py

```diff
@@ -637,12 +669,16 @@ def generate_main_class(endpoints):
             try {
               int i = 0;
               for (String id: ids) {
-                keys[i] = clz.newInstance();
+                keys[i] = clz.getConstructor().newInstance();
                 keys[i].name = id;
                 keys[i].type = keyType;
                 i++;
               }
-            } catch (InstantiationException | IllegalAccessException e) { /* can't be */ }
+            }
+            catch (InstantiationException | IllegalAccessException e) { /* can't be */ }
+            catch (NoSuchMethodException | InvocationTargetException e) {
+              e.printStackTrace();
```

Comment:
Eating exception? I think it is better to propagate it up to signal that something is wrong.